src/cocotb/simtime.py

```diff
@@ -17,28 +17,48 @@
 from typing import Literal, cast, overload
 
 from cocotb import simulator
-from cocotb._typing import RoundMode, TimeUnit
 
 if sys.version_info >= (3, 10):
     from typing import TypeAlias
 
 __all__ = (
+    "RoundMode",
+    "TimeUnit",
     "convert",
     "get_sim_time",
     "time_precision",
 )
 
+RoundMode: TypeAlias = Literal["error", "round", "ceil", "floor"]
+"""
+How to handle non-integral step values when quantizing to simulator time steps.
+
+One of ``'error'``, ``'round'``, ``'ceil'``, or ``'floor'``.
+
+When *round_mode* is ``"error"``, a :exc:`ValueError` is thrown if the value cannot
```

Comment:
Should we rewrite the description of these types so that it also works from the POV of a 3rd party library, e.g. like "cocotb throws"?